adaptivefiltering/utils.py

```diff
@@ -15,3 +22,52 @@ def stringify_value(value):
         return " ".join(value)
 
     return str(value)
+
+
+def trim(path):
+    """Trim the whitespace around pictures from matplotlib"""
+
+    im = Image.open(path)
+    bg = Image.new(im.mode, im.size, im.getpixel((0, 0)))
+    diff = ImageChops.difference(im, bg)
+    diff = ImageChops.add(diff, diff, 2.0, -100)
+    bbox = diff.getbbox()
+    if bbox:
+        im = im.crop(bbox)
+        im.save(path)
+
+
+def convert_picture_to_base64(path):
+    image = Image.open(path)
+    ext = os.path.splitext(path)[1][1:]  # file extension
+    f = BytesIO()
+    image.save(f, ext)
+    data = b64encode(f.getvalue())
+    data = data.decode("ascii")
+    url = "data:image/{};base64,".format(ext) + data
+    return url
+
+
+def convert_Segmentation(segmentation, srs_out, srs_in="EPSG:4326"):
```

Comment:
lower case `s` for consistency.  Also: As this one directly operates on `Segmentation` objects, shouldn't it live in `segmentation.py`?